src/engine_plugin.cpp

```diff
@@ -136,13 +136,22 @@ class engine_plugin_impl : std::enable_shared_from_this<engine_plugin_impl> {
          return silkworm::db::read_canonical_header(txn, head_num);
       }
 
-      std::optional<silkworm::Block> get_head_block() {
-         auto header = get_head_canonical_header();
-         if(!header) return {};
+      std::optional<silkworm::Block> get_canonical_block_at_height(const std::optional<uint64_t>& override_height) {
```

Comment:
override_height => height

Reply:
Also for small types (uint64_t) we can pass std::optional by value.
```c++
std::optional<silkworm::Block> get_canonical_block_at_height(std::optional<uint64_t> height)
```

Reply:
done